gobblin-api/src/main/java/org/apache/gobblin/publisher/SingleTaskDataPublisher.java

```diff
@@ -75,4 +75,8 @@ public static SingleTaskDataPublisher getInstance(Class<? extends DataPublisher>
 
     return (SingleTaskDataPublisher) DataPublisher.getInstance(dataPublisherClass, state);
   }
+
+  public boolean submitSingleTaskLineageEvent (WorkUnitState state) {
```

Comment:
Ditto

Reply:
@ibuenros Add the aggregation logic in LineageInfo, removed task level lineage event because we are not using this now.